full/src/main/java/apoc/json/JsonExtended.java

```diff
@@ -0,0 +1,29 @@
+package apoc.json;
+
+import apoc.Extended;
+import apoc.result.StringResult;
+import apoc.util.JsonUtil;
+import org.neo4j.procedure.Context;
+import org.neo4j.procedure.Description;
+import org.neo4j.procedure.Name;
+import org.neo4j.procedure.Procedure;
+
+import java.util.List;
+import java.util.stream.Stream;
+
+
+@Extended
+public class JsonExtended {
+
+    @Context
+    public org.neo4j.graphdb.GraphDatabaseService db;
+
+    @Procedure
```

Comment:
wouldn't it be better if this was a function that returned a list, so that people could just say  `WHERE size(apoc.json.validate(json)) = 0` ...